src/index.html

```diff
@@ -1,17 +1,187 @@
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+      <fieldset class="inputs-block">
+        <legend>Personal information</legend>
+
+        <label class="input-row">
+          Surname:
+
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="input-row">
+          Name:
+
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="input-row">
+          How old are You?
+
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="input-row">
+          Full date of birth:
+
+          <input type="date" name="birthday">
+        </label>
+
+        <label class="input-row">
+          I accept the term of the agreement
+
+          <input
+            type="checkbox"
+            name="agreement"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="inputs-block">
+        <legend>Registration</legend>
+
+        <label class="input-row">
+          E-mail:
+
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="input-row">
+          Password:
+
+          <input
+            type="password"
+            name="password"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="inputs-block">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="input-row">
+          Do you love cats?
+
+          <label>
+
+            <input
+              type="radio"
+              name="cats"
+              value="yes"
+            >
+
+            Yes
+          </label>
+
+          <label>
+
+            <input
+              type="radio"
+              name="cats"
+              value="no"
+            >
+
+            No
+          </label>
+        </div>
+
+        <label class="input-row">
+          What is your favorite color?
+
+          <input type="color" name="color">
+        </label>
+
+        <label class="input-row">
+          What time do you go to bed?
+
+          <input
+            type="time"
+            name="time"
+            step="1"
+          >
+        </label>
+
+        <label class="input-row">
+          What are your favorite brand of cars?
+
+          <select multiple name="cars">
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="input-row">
+          How do you rate our work?
+
+          <input
+            type="range"
+            name="rate"
+            minlength="0"
```

Comment:
Input type `range` should have `min` and `max` attributes rather than `minlength` and `maxlength`, these are more appropriate for inputs with text, for example you want you password to be of some minimal length.

Reply:
Okay :). Thank you. I hope now everything is correct :-)